### PR TITLE
Minor: replace `bit_util::ceil()` to `div_ceil()`

### PR DESCRIPTION
**Pull request by haohuaijin**
## Which issue does this PR close?

Closes #.

## Rationale for this change

the `div_ceil()` is already stable.

### CODE REVIEW

**Comment by tustvold** [comment]
FYI I believe this will implicitly bump the MSRV of datafusion to the latest of 1.73 - I'm not sure what the policy on this is

File: datafusion-cli/Cargo.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ keywords = ["arrow", "datafusion", "query", "sql"]
 license = "Apache-2.0"
 homepage = "https://github.com/apache/arrow-datafusion"
 repository = "https://github.com/apache/arrow-datafusion"
-rust-version = "1.70"
+rust-version = "1.73"
```
Comment (tustvold) [inline datafusion-cli/Cargo.toml:28]: It is fairly standard practice to set the MSRV to accommodate at least a couple of older versions, I'm not sure what DataFusion's policy is, but updating to 1.73 which was only released 9 days ago would be a very aggressive posture.

For comparison arrow-rs has an MSRV of 1.62

Comment (haohuaijin) [reply datafusion-cli/Cargo.toml:28]: agree

**Comment by alamb** [comment]
So perhaps we should sit on this PR for some time (6 months?) until 1.73 has been out for a longer time.

**Pull request closed without merging**